homeassistant/components/mazda/button.py

```diff
@@ -29,7 +37,14 @@ async def handle_button_press(
 
     try:
         await api_method(vehicle_id)
-    except Exception as ex:
+    except (
+        MazdaException,
+        MazdaAuthenticationException,
+        MazdaAccountLockedException,
+        MazdaTokenExpiredException,
+        MazdaAPIEncryptionException,
+        MazdaLoginFailedException,
+    ) as ex:
```

Comment:
This could also be a constant if you need to use it in other places

Reply:
Ok, I'll keep that in mind.